config/default.js

```diff
@@ -222,4 +222,13 @@ module.exports = {
       amount: 200,
     },
   ],
+
+  // for Browser use
+  optimistUrl:
```

Comment:
Can we reuse 
https://github.com/EYBlockchain/nightfall_3/blob/3f787ae85652dc739f69c1b823c42952bcd9d42f/config/default.js#L170

or 

https://github.com/EYBlockchain/nightfall_3/blob/3f787ae85652dc739f69c1b823c42952bcd9d42f/config/default.js#L170

Reply:
These endpoints have been remanded to `proposerUrl` to reflect what they are actually connecting to. Locally this still points to `optimist` but will need to move to `proposer` when we have a more mature `proposer` application